### PR TITLE
To support CA setting when baselinecaps is none

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As title, to address this https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/32634/rehearse-32634-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-aws/1575797602731954176

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
cc: @jinyunma

File: ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh [hunk 1]
```diff
@@ -130,12 +130,12 @@ function check_mirror_registry () {
 }
 
 set_proxy
-run_command "oc get config.samples.operator.openshift.io cluster"; ret=$?
+ret=0
+run_command "oc get config.samples.operator.openshift.io cluster" || ret=$?
```
Comment (jinyunma) [inline ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:134]: From failed job, I didn't see any echo message from `if` or `else` block.
It seems that failure is in function run_command when running `run_command "oc get config.samples.operator.openshift.io cluster"`, the update could not fix the issue in failed job.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/32634/rehearse-32634-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-aws/1575797602731954176/artifacts/installer-rehearse-aws/set-sample-operator-disconnected/build-log.txt

Comment (jianzhangbjz) [reply ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:134]: Hi @jinyunma , I guess you checked the wrong job, the above job ran on Sep. 30. The `||` will address this issue.
image

Comment (jinyunma) [reply ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:134]: get it, thanks for checking.

**Comment by xiuwang** [comment]
Thanks, Jian
/lgtm

**Pull request merged**